.github/workflows/dev-ci.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
       - name: Run tests
         env:
           SQLALCHEMY_DATABASE_URI: postgresql://postgres:postgres@localhost/test_notification_api
-        run: /bin/bash -c "pip install -r requirements-app.txt && pip install -r requirements_for_test.txt && make test"
```

Comment:
This is the mistake from #644.  All containers that run non-locally should use requirements.txt, which freezes all requirements.  Testing a container built from requirements-app.txt is testing a container that likely uses newer package versions than what is currently in any AWS environment.